<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepositExchangeContractDialog</class>
 <widget class="QDialog" name="DepositExchangeContractDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>580</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <widget class="QWidget" name="containerWidget" native="true">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>500</width>
      <height>320</height>
     </rect>
    </property>
    <widget class="QToolButton" name="cancelBtn">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>250</y>
       <width>96</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QToolButton" name="okBtn">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>250</y>
       <width>96</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QComboBox" name="assetComboBox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>54</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>充值数量</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="amountLineEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>150</y>
       <width>240</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>当前账户</string>
     </property>
    </widget>
    <widget class="QLabel" name="accountNameLabel">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>充值币种</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>账户余额</string>
     </property>
    </widget>
    <widget class="QLabel" name="balanceLabel">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>110</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="tipLabel">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>230</y>
       <width>200</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>190</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>输入钱包密码：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="pwdLineEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>190</y>
       <width>240</width>
       <height>30</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="closeBtn">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>16</y>
       <width>18</width>
       <height>18</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
